marketplace-service/src/test/java/com/axonivy/market/service/impl/ProductServiceImplTest.java

```diff
@@ -623,13 +623,13 @@ private GHCommit mockGHCommitHasSHA1(String sha1) {
 
   private GHContent mockGHContentAsMetaJSON() {
     var mockGHContent = mock(GHContent.class);
-    when(mockGHContent.getName()).thenReturn(META_FILE);
+    lenient().when(mockGHContent.getName()).thenReturn(META_FILE);
```

Comment:
remove using of lenient. From my POV, we only setup mock stub if needed and try to avoid mocking as much as posible.